vendor/autoload.php

```diff
@@ -2,6 +2,6 @@
 
 // autoload.php @generated by Composer
 
-require_once __DIR__ . '/composer' . '/autoload_real.php';
+require_once __DIR__ . '/composer/autoload_real.php';
```

Comment:
I "cheat" and do reverse searches for "openemr_composer" in my terminal history to run this all the time:

```composer install && ./vendor/bin/phing vendor-clean && rm -rf vendor/phing && rm -rf vendor/bin/phing && composer dump-autoload -o # openemr_composer```

This is what I used to check in this PR. Is something off with the command?